x-pack/plugins/apm/server/saved_objects/apm_telemetry.ts

```diff
@@ -16,4 +17,12 @@ export const apmTelemetry: SavedObjectsType = {
     dynamic: false,
     properties: {},
   },
+  modelVersions: {
```

Comment:
Is it worth having the right schema here? Same with https://github.com/elastic/kibana/pull/159881/files#diff-efb004ee8ddf2f202d9a423ef9f193935bbf2785a9068120ccd18cac5c006fe3R33